File: soma1bi.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    long long tot = 0;
    long long n = 1000000000;
    clock_t ti = clock();
    for (long long i = 1; i <= n; i++) {
        tot += i;
    }
    clock_t tf = clock();
    double dt = (double)(tf-ti)/CLOCKS_PER_SEC;
    printf("Soma: %lld\n", tot);
    printf("Tempo: %.15lf\n", dt);
    return 0;
}

File: soma1bi.lua
function soma()
    local tot = 0
    local n = 1000000000
    for i = 1, n do
        tot = tot + i
    end
    return tot
end
local ti = os.clock()
local res = soma()
local tf = os.clock()
local dt = tf-ti
print("Soma:", res)
print("Tempo:", string.format("%.15f", dt))